pkg/controller/master/migration/vmim_controller.go

```diff
@@ -222,3 +228,19 @@ func isAbortRequest(vmim *kubevirtv1.VirtualMachineInstanceMigration) bool {
 	}
 	return false
 }
+
+// https://github.com/harvester/harvester/issues/6193
+// The UI VM page is anchored on VM object, only when VM object's resorceVersion is changed
+// then UI will update the VM action menu
+// VM's migration is done mainly between vmi and vmimm
```

Comment:
What is vmimm ?

Reply:
should be `vmim` for `VirtualMachineInstanceMigration`, let me update. thanks.